pwndbg/commands/got.py

```diff
@@ -30,14 +28,19 @@ def got(name_filter=''):
     relro_status = pwndbg.wrappers.checksec.relro_status()
     pie_status = pwndbg.wrappers.checksec.pie_status()
     jmpslots = list(pwndbg.wrappers.readelf.get_jmpslots())
-
     if not len(jmpslots):
-        print(red("NO JUMP_SLOT entries available in the GOT"))
+        print(message.error("NO JUMP_SLOT entries available in the GOT"))
         return
+
     if "PIE enabled" in pie_status:
         bin_text_base = pwndbg.memory.page_align(pwndbg.elf.entry())
 
-    print("\nGOT protection: %s | GOT functions: %d\n " % (green(relro_status), len(jmpslots)))
+    relro_color = message.off
+    if 'Partial' in relro_status:
```

Comment:
Hope there isn't any edge case ;f

Reply:
well thats how we stricty use  our wrapper/checksec.py so it must be the same output :yum: